app/models/vm_or_template/operations/snapshot.rb

```diff
@@ -65,6 +65,7 @@ def raw_remove_snapshot(snapshot_id)
     begin
       run_command_via_parent(:vm_remove_snapshot, :snMor => snapshot.uid_ems)
     rescue => err
+      create_notification(:vm_snapshot_failure, :error => err.to_s, :snapshot_op => "remove")
```

Comment:
Don't you need a subject here?

Reply:
`create_notification` method (https://github.com/ManageIQ/manageiq/blob/master/app/models/vm_or_template.rb#L1782) does that.